geoips/commandline/cmd_instructions.py

```diff
@@ -32,23 +32,26 @@ def cmd_instructions_modified(ancillary_dirname):
     ----------
     ancillary_dirname: str
         - The path to the folder which contains the help instructions for the CLI
+    fname: str (optional)
+        - The name of the file (no extension) that we will be checking the modification
+          time of.
 
     Returns
     -------
     yaml_recently_modified: bool
         - The truth value as to whether or not the yaml cmd_instructions were modified
           more recently than the json cmd_instructions
     """
-    json_mtime = getmtime(f"{ancillary_dirname}/cmd_instructions.json")
-    yaml_mtime = getmtime(f"{ancillary_dirname}/cmd_instructions.yaml")
+    json_mtime = getmtime(f"{ancillary_dirname}/{fname}.json")
+    yaml_mtime = getmtime(f"{ancillary_dirname}/{fname}.yaml")
     yaml_recently_modified = False
-    if yaml_mtime > json_mtime:
+    if yaml_mtime >= json_mtime:
         # yaml file was modified more recently than json_mtime
         yaml_recently_modified = True
     return yaml_recently_modified
 
 
-def get_cmd_instructions(ancillary_dirname=None):
+def get_instructions(ancillary_dirname=None, fname="cmd_instructions"):
```

Comment:
I would prefer that `fname` takes a full filename including the extension. I think we should use `.yaml` here since that is what is normally referenced. The `.json` files are just an under-the-hood efficiency update.

Reply:
Changed this to a full name as in ``cmd_instructions.yaml``